src/components/OpenWith/IconFileMap.js

```diff
@@ -0,0 +1,15 @@
+import IconAdobeSign from 'box-react-ui/lib/icons/adobe-sign/IconAdobeSign';
```

Comment:
This will be reused by the dropdown. Eventually this may be removable depending on API changes.